lists/pe/RUC.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": {
+    "en": "National Superintendency of Customs and Tax Administration (Peru)",
```

Comment:
I think we need:

"en": "RUC Number (Peru)"
"local": "Registro Único del Contribuyente (Perú)"